core/sdk/src/main/java/com/mongodb/stitch/core/internal/net/OkHttpTransport.java

```diff
@@ -105,6 +107,10 @@ private static Response handleResponse(final okhttp3.Response response) {
   @Override
   // This executes a request synchronously
   public Response roundTrip(final Request request) throws IOException {
+    if (request.getBody() != null && request.getBody().length >= MAX_REQUEST_SIZE) {
```

Comment:
[discussion] I know this was discussed offline, but thinking more about this, I don't think we should actually enforce this at the SDK level. If the server decides to increase the request size limit, it doesn't seem like the client should need to be updated to take advantage of that. Thoughts @dkaminsky ?

Reply:
Resolved on slack that we would like to keep this